### PR TITLE
Move utility

### PR DESCRIPTION
**Pull request by jgeorge37**
Move utility out of the api

### CODE REVIEW

File: acmw-db-app/test/utility/utility.test.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import {
     validateLastNameDotNum, 
     validateTime,
     validateName
-} from '../../../pages/api/utility';
+} from '../..//utility/utility';
```
Comment (SnobbyDragon) [inline acmw-db-app/test/utility/utility.test.js:10]: there's an extra `/`

**Review by SnobbyDragon — CHANGES_REQUESTED** [review]
ngl im not really a fan of the folder having the same name as the file but it's fine. idk if an extra `/` would case probblmes

**Comment by jgeorge37** [comment]
Files and folders having the same name is very common

**Review by SnobbyDragon — APPROVED** [review]
lgtm!

**Comment by SnobbyDragon** [comment]
> Files and folders having the same name is very common

that doesn't mean ill like it 😤  but approved

**Pull request merged**